### DISCUSSION TITLE
Attached files are not shown

### DISCUSSION THREAD

**Discussion by Oguret2** [slatinsky/DiscordChatExporter-frontend#46] [Q&A]
Attached files are not displayed. There are no problems with attached photos or videos, but other files are not displayed at all. There are files in the attachment folder and they are listed in the chat json. Add the ability to display files of any format plz.
![yAwWN0NKak](https://github.com/slatinsky/DiscordChatExporter-frontend/assets/152782570/779742ad-603d-4ec5-8f61-7b6668dd22ff)
![bGZRmTWGG9](https://github.com/slatinsky/DiscordChatExporter-frontend/assets/152782570/54032402-3f47-4f4f-a2f2-7698b76f5b95)

**Comment by slatinsky** [comment]
Transfered to an issue #47
